.changeset/fluffy-points-think.md

```diff
@@ -0,0 +1,11 @@
+---
+"@comet/blocks-admin": minor
+---
+
+Add Support for Icons in OneOfBlock tabs
+
+Usage:
+
+-   Add tabLabels to `createOneOfBlock` props
+    -   `tabLabels: { image: <Image />, damVideo: <Video />, youTubeVideo: <YouTube /> }`
+    -   the keys in the tabLabels need to be the same as the `supportedBlocks`
```

Comment:
It's kinda odd that the changeset mentions icons, but the option is called `tabLabels`. Should we adapt the changeset?